src/pages/index.astro

```diff
@@ -1,68 +1,50 @@
 ---
 import Layout from '../layouts/Layout.astro';
-import Card from '../components/Card.astro';
+import {SITE_TITLE} from "../constants";
 ---
 
-<Layout title="Welcome to Astro.">
+<Layout title="Home">
 	<main>
+		<div class="hero min-h-screen">
+			<div class="hero-content text-center">
+				<div class="max-w-ld flex flex-col items-center">
+					<h1 class="text-5xl pb-3 font-bold">This Is Learning Conference</h1>
+					<h2 class="text-3xl font-bold">Happening On 16-17 Of May</h2>
+					<p class="py-6">Registration Opens On Monday 19th Feb 6am CET</p>
+					<a href="https://konfhub.com/tc-b34e332d" target="_blank"><button class="btn btn-primary text-2xl font-medium">Register</button></a>
+
+					<div class="stats shadow mt-8 grid-flow-row md:grid-flow-col">
```

Comment:
Due to the current state, we should comment this part as we only know the track part (4 tracks).